pgml-dashboard/build.rs

```diff
@@ -3,7 +3,6 @@ use std::process::Command;
 
 fn main() {
     println!("cargo:rerun-if-changed=migrations");
-    println!("cargo:rerun-if-changed=src");
```

Comment:
@chillenberger This rebuilt the app every time we changed css which was kinda slow. Instead, I'm running:

```
cargo watch  --exec 'pgml-components bundle'  --watch src/  --watch static/  --ignore bundle.*.*
```

in a separate terminal.